packages/ai-bot/README.md

```diff
@@ -0,0 +1,46 @@
+# AI Bot
+
+This is an experimental AI assistant for boxel using GPT4.
+
+Applications can communicate with this via matrix chat rooms, it will automatically join any room it is invited to on the server.
+
+Access to the matrix server currently equates to access to GPT4 if this bot is connected.
+
+## Setup
+
+### Matrix
+
+This bot requires a matrix user to connect as. Create one as described in the matrix package documentation with the username `aibot`. The bot will default to trying to connect with the password `pass` but you can choose your own and set the `BOXEL_AIBOT_PASSWORD` environment variable.
+
+It will default to connecting on `http://localhost:8008/`, which can be overridden with the `MATRIX_URL` environment variable.
+
+### Boxel Development
+
+If you are working on boxel development, you do not need to connect this up to OpenAI. 
+
+### Access to GPT4
+
+You can get an OpenAI api key one from the staging parameter store or ask within the team.
+Set this as the `OPENAI_API_KEY` environment variable. Note that if you set this broadly (such as in your bashrc) this will be used by default for many other openai based tools.
+
+    OPENAI_API_KEY="sk-..."
+
+## Running
+
+Start the server with `pnpm start` or `pnpm start-dev` for live reload.
```

Comment:
`pnpm start` gets an error 

```
import { CardResource } from './card-document';
^^^^^^

SyntaxError: Cannot use import statement outside a module
    at Object.compileFunction (node:vm:360:18)
    at wrapSafe (node:internal/modules/cjs/loader:1049:15)
    at Module._compile (node:internal/modules/cjs/loader:1084:27)
    at Module._extensions..js (node:internal/modules/cjs/loader:1174:10)
    at Object.require.extensions.<computed> [as .ts] (/Users/justinthong/Github/boxel/node_modules/.pnpm/ts-node@10.9.1_@types+node@18.15.13_typescript@5.1.6/node_modules/ts-node/src/index.ts:1608:43)
    at Module.load (node:internal/modules/cjs/loader:998:32)
    at Function.Module._load (node:internal/modules/cjs/loader:839:12)
    at Module.require (node:internal/modules/cjs/loader:1022:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (/Users/justinthong/Github/boxel/packages/ai-bot/setup-logger.ts:1:1)
```

Reply:
Probably missing runtime-common as a dependency?

Reply:
yeah could be, this did work before and pnpm start-dev works :/